build/generateIcons.js

```diff
@@ -0,0 +1,62 @@
+
+try {
+    console.log(' ');
+    console.log(' -------------------------------------------------------------------------------');
+    console.log('  ICONS GENERATOR - PROCESS START');
+    console.log(' -------------------------------------------------------------------------------');
+    console.log(' ');
+    const webfontsGenerator = require('vusion-webfonts-generator');
+    const fs = require('fs');
+    const path = require('path');
+    const svgDirectory = path.join(__dirname, '../web/client/themes/default/svg/');
+    const files = fs.readdirSync(svgDirectory);
+    const destinationDirectory = path.join(__dirname, '../web/client/themes/default/');
+    const dest = path.join(destinationDirectory, 'icons');
+    const cssDest = path.join(destinationDirectory, 'icons.less');
+    const cssTemplate = path.join(destinationDirectory, 'icons.template.hbs');
+    console.log(`  Template -> ${cssTemplate}`);
+    console.log(' ');
+    console.log(' ');
+    webfontsGenerator({
+        files: files.map(function(file) {
+            return `${svgDirectory}${file}`;
+        }),
+        dest: dest,
+        cssDest: cssDest,
+        fontName: 'icons',
+        templateOptions: {
+            classPrefix: 'glyphicon-',
+            baseSelector: '.glyphicon'
+        },
+        cssTemplate: cssTemplate
+    }, function(error) {
+        if (error) {
+            console.log(error);
```

Comment:
maybe i would log the error after the title

or you can add something to the end